addon/components/nrg-render-template-block/component.js

```diff
@@ -0,0 +1,58 @@
+import Component from '@ember/component';
+import layout from './template';
+
+export default Component.extend({
+  classNames: ['ui hidden message'],
```

Comment:
I wouldn't use these classes since they have intended meaning. 

Please write a class for just `display: none`.